### DISCUSSION TITLE
WeatherKit会支持Shadowrocket吗

### DISCUSSION THREAD

**Discussion by cohirogaobo** [NSRingo/.github#11] [Q&A 问答] [answered]
希望WeatherKit可以支持Shadowrocket，现在支持的几款软件安装门槛和专业性都比较高，感觉上手有难度。

**Comment by VirgilClyne** [comment, marked as answer]
不会，Shadowrocket 所使用的 JavaScriptCore 引擎缺乏必要的 Web API 组件
